Readability.js

```diff
@@ -26,8 +26,8 @@
  * @param {Object}       options The options object.
  */
 function Readability(doc, options) {
-  // In some older versions, people passed a URI object as the first argument. Cope:
-  if (doc && !doc.documentElement && doc.spec) {
+  // In some older versions, people passed a URI object or string as the first argument. Cope:
+  if (doc && ((!doc.documentElement && doc.spec) || typeof doc == "string")) {
```

Comment:
Perhaps it would be worth fixing #441 at the same time and doing something like the following?

```js
if (doc && doc.documentElement) {
  // ok
} else if (options && options.documentElement) {
  doc = options;
  options = arguments[2];
} else {
  throw new Error("invalid arguments");
}
```

I think some of the tests pass in an empty object for the document though, so those would also need to be changed.

Reply:
Good point. Also thanks for the heads-up about the test!

I'll hold off printing a warning for now. Maybe when we publish to npm...